Makefile

```diff
@@ -151,14 +151,12 @@ else ifeq ($(TYPE),release-linux-gnu)
 # linked to avoid depending on the target's available libstdc++.
 XHOST_TRIPLE := x86_64-unknown-linux-gnu
 override LINKFLAGS += -s -w -extldflags "-static-libgcc -static-libstdc++"
-override GOFLAGS += -installsuffix release-gnu
```

Comment:
I may have hit approve too fast. What do these do and why could they be removed?

Reply:
So before Go 1.10, Go would write installed assets to `GOPATH/pkg/GOOS_GOARCH`. With a race build, it would write to `GOPATH/pkg/GOOS_GOARCH_race`. Similarly for `msan` builds. You can tack on your own suffix by setting `-installsuffix`; `go build -installsuffix foo` would install into `GOPATH/pkg/GOOS_GOARCH_foo`, for example.

We previously had to tap into this feature when building Linux release builds, because `go build -extldflags "-static-libgcc -static-libstdc++"` (GNU builds) and `go build -extldflags "-static"` (musl builds) would _both_ write into `$GOPATH/pkg/linux_amd64` and clobber each other's artifacts. Since Go 1.10, though, Go has a much smarter package cache that lives outside `$GOPATH/pkg`, so we don't need to do this micromanagement.

Reply:
:+1: